### DISCUSSION TITLE
Outdated doc

### DISCUSSION THREAD

**Discussion by jaxvanyang** [zed-industries/zed#13440] [Feature Requests]
I noticed that the doc at https://zed.dev/docs/configuring-zed#soft-wrap is different to the source code at https://github.com/zed-industries/zed/blob/78bc3a9a369cf6d9d60f29d9a7850259015af9e5/assets/settings/default.json#L480-L491

I have to say that the default value is not correct in the doc confused me. Didn't check if there is other difference, just let you know the doc need update.

**Comment by SomeoneToIgnore** [comment]
Definitely out of sync, I've filed the https://github.com/zed-industries/zed/pull/13457 to fix it, thank you.
